lib/drivers/medtronic600Driver.js

```diff
@@ -1332,8 +1716,91 @@ module.exports = (config) => {
     processData(progress, data, cb) {
       debug('in processData');
       cfg.builder.setDefaults({
-        deviceId: driver.pumpSession.bcnlModelAndSerial,
+        deviceId: `${data.settings.pumpModel}:${data.settings.pumpSerial}`,
       });
+
+      // TODO - add more time change records
```

Comment:
Just a note, as I notice there is a TODO: Until the time change records are added, UTC bootstrapping will not work as expected.